### PR TITLE
jsHint fixes

### PR DESCRIPTION
**Pull request by zalun**
Alert if Add-ons Builder helper is not the current one

WARNING: switch checkAddonHelperVersion after addon Add-pons Builder
Helper will respond with version number to
window.mozFlightDeck.send({cmd: 'version'});

fixes https://bugzilla.mozilla.org/show_bug.cgi?id=687138

### CODE REVIEW

File: media/base/js/FlightDeck.js [hunk 1]
```diff
@@ -338,9 +340,27 @@ var FlightDeck = new Class({
     },
 
     /*
+     * Method: alertIfOldHelper
+     * Show a warning if old add-on builder helper is installed
+     */
+    alertIfOldHelper: function() {
+        if (settings.addons_helper_version && window.mozFlightDeck) {
+            response = window.mozFlightDeck.send({cmd: 'version'});
```
Comment (seanmonstar) [inline media/base/js/FlightDeck.js:349]: leaked global `response`

File: media/base/js/FlightDeck.js [hunk 2]
```diff
@@ -338,9 +340,27 @@ var FlightDeck = new Class({
...
+            if (!response.success || response.msg < settings.addons_helper_version) {
+                fd.warning.alert(
+                        'Upgrade Add-on Builder Helper',
+                        'There is a newer version ({addons_helper_version}) available.<br/> Please install <a href="{addons_helper}">the current one</a>.'.substitute(settings));
+            } else alert('WTF?')
```
Comment (seanmonstar) [inline media/base/js/FlightDeck.js]: are we sure we want to popup an alert yelling at the user "WTF" ? lol

Comment (clouserw) [inline media/base/js/FlightDeck.js]: heh, plz no

**Comment by zalun** [comment]
I hope it's fine now

**Pull request merged**